ElastosBountyProgram/back-end/scripts/emp35/task_rewards_ela_to_usd.js

```diff
@@ -0,0 +1,24 @@
+var conversionRate = 6.82;
+
+db.tasks.find({ "reward.isUsd": false }).forEach(function (task) {
+		db.tasks.update(
+			{ "_id": task._id }, { $set: 
+		  	{
+	  			"reward.usd": (task.reward.ela * conversionRate),

```

Comment:
Some tasks might not have rewards at all, so I would ```task.reward.ela || 0``` here just in case (same in rewardUpFront)